### ISSUE TITLE
Enhancement Idea:  display/hide SKU

### ISSUE THREAD

**Issue by MadSpaniard** [jameskoster/woocommerce-product-details-customiser#5]
Hello - I thought it might be helpful to add SKU as one of the items that can be hidden using this plugin.  I tried inserting the following code snippets to "details-customiser.php" and these seem to work:

Inserted this on line #52:

<pre>
array(
    'desc'      => __( 'Product SKU', 'woocommerce-product-details-customiser' ),
    'id'        => 'wc_pdc_sku',
    'type'      => 'checkbox',
    'default'   => 'yes'
),
</pre>

Inserted this on line #98:

<pre>
add_option( 'wc_pdc_sku', 'yes' );
</pre>

Inserted this on line #172:

<pre>
// SKU
if ( get_option( 'wc_pdc_sku' ) == 'no' ) {
    add_filter( 'wc_product_sku_enabled', '__return_false' );
}
</pre>

Do you think something of this ilk may be worth including in your plugin?

Thanks as always.

**Comment by jameskoster** [comment]
Doesn't the `wc_product_sku_enabled` filter disable the SKU globally, not just the display?

**Comment by MadSpaniard** [comment]
You are correct... well, the suggestion for "hide sku" may have had a worthy intent, but not so worthy implementation... Of course, a CSS tweak or an adjustment to the underlying template file could achieve the desired result, but was trying to come up with a more dynamic solution that could be easily toggled... maybe the CSS tweak is the next best thing for now.

Thanks for looking into this, best regards.

**Comment by jameskoster** [comment]
Yeah I don't really have time to add this myself right now. Do feel free to send a PR if you want to investigate further though :-)
